src/factories/CodeFlowFactory.ts

```diff
@@ -77,14 +77,14 @@ export namespace CodeFlowFactory {
      * @param sarifCodeFlows Used if a codeflow needs to be remapped
      * @param runId used for mapping uribaseids
      */
-    export async function tryRemapCodeFlows(explorerController: ExplorerController, codeFlows: CodeFlow[], sarifCodeFlows: sarif.CodeFlow[], runId: number): Promise<void> {
+    export async function tryRemapCodeFlows(fileMapper: FileMapper, runInfo: RunInfo, codeFlows: CodeFlow[], sarifCodeFlows: sarif.CodeFlow[], runId: number): Promise<void> {
         for (const [cFKey, codeFlow] of codeFlows.entries()) {
             for (const [tFKey, threadFlow] of codeFlow.threads.entries()) {
                 for (const [stepKey, step] of threadFlow.steps.entries()) {
-                    if (step.location && step.location.mapped) {
+                    if (step.location && !step.location.mapped) {
```

Comment:
>!step.location.mapped [](start = 41, length = 21)

Yes, this was a regression I caught and verified manually as well as the original code (which I kept a copy of)